packages/nextjs/src/server/clerkMiddleware.ts

```diff
@@ -187,7 +187,12 @@ const parseHandlerAndOptions = (args: unknown[]) => {
   ] as [ClerkMiddlewareHandler | undefined, ClerkMiddlewareOptions];
 };
 
-export const createAuthenticateRequestOptions = (clerkRequest: ClerkRequest, options: ClerkMiddlewareOptions) => {
+type AuthenticateRequest = Pick<ClerkClient, 'authenticateRequest'>['authenticateRequest'];
```

Comment:
This was taken from the Astro SDK to fix a TypeScript error where it was unable to build declaration files for some reason that I'm honestly not smart enough to properly figure out. Basically it wanted an explicit return type instead of an inferred one.